rest/models/pom.xml

```diff
@@ -51,10 +51,6 @@
 
     <build>
         <plugins>
-            <plugin>
-                <groupId>org.apache.servicemix.tooling</groupId>
-                <artifactId>depends-maven-plugin</artifactId>
-            </plugin>
```

Comment:
@ahoffer https://github.com/connexta/ion-query-api/pull/12#pullrequestreview-277939648

It was used to generate a dependency report that wasn't being used: https://github.com/connexta/ion-ingest-api/pull/38/files#diff-600376dffeb79835ede4a0b285078036L556-L568. I missed deleting this plugin in this pom in that PR. If we want to start using the report, we can add it back in.

Reply:
Thanks for the explanation!